Cabal-syntax/src/Distribution/Compat/Lens.hs

```diff
@@ -221,10 +239,10 @@ cloneLens l f s = runPretext (l pretextSell s) f
 -------------------------------------------------------------------------------
 
 -- | @lens@ variant is also parametrised by profunctor.
-data Pretext a b t = Pretext { runPretext :: forall f. Functor f => (a -> f b) -> f t }
+data Pretext a b t = Pretext {runPretext :: forall f. Functor f => (a -> f b) -> f t}
```

Comment:
Are blanks after { and before } configurable? Perhaps, I'd prefer having them.

Reply:
Uh good question. @brandonchinn178 Do you think this is something that could be easily added by a newcomer?

Reply:
Sure thing! We already have `record-brace-space` for a space between constructor and brace, it's just a matter of adding an option for a space between brace and brace contents. Shouldnt be too difficult